cmd/lit-af/chancmds.go

```diff
@@ -106,6 +108,15 @@ func (lc *litAfClient) FundChannel(textArgs []string) error {
 	if err != nil {
 		return err
 	}
+
+	if len(textArgs) > 4 {
+		data, err := hex.DecodeString(textArgs[4])
+		if err != nil {
+			return err
+		}
+		copy(args.Data[:], data[:32])
```

Comment:
This crashes lit-af, `panic: runtime error: slice bounds out of range`

Can't assume it's 64 hex characters... but also, why assume it's hex at all?  Maybe just look at the len() of textArgs[4] and truncate if it's more than 32?